pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[tool.poetry]
+name = "lda"
+version = "2.1.0"
+description = "Topic modeling with latent Dirichlet allocation"
+authors = ["lda developers <lda-users@googlegroups.com>"]
+license = "MPL 2.0"
+readme = "README.rst"
+
+[tool.poetry.build]
+generate-setup-file = true
+script = "build.py"
+
+[tool.poetry.dependencies]
+python = "<3.13,>=3.9"
+numpy = "^1.26.0"
+
+[tool.poetry.group.dev.dependencies]
+black = "^23.9.1"
+isort = "^5.12.0"
+pre-commit = "^3.4.0"
+ruff = "^0.0.292"
+
+[tool.poetry.group.test.dependencies]
+scipy = "^1.11.3"
+
+[tool.poetry.group.build.dependencies]
+cibuildwheel = "^2.16.2"
+
+[tool.black]
+line-length = 99
+
+[tool.isort]
+line_length = 99
+use_parentheses = true
+include_trailing_comma = true
+multi_line_output = 3
+
+[tool.ruff]
+line-length = 99
+
+[tool.ruff.per-file-ignores]
+"__init__.py" = ["F401"]
+
+[tool.cibuildwheel]
+build = "cp39-* cp310-* cp311-* cp312-*"
+skip = ["pp*", "*i686", "*win32"]
```

Comment:
We skip these because there are currently some issues, see: https://github.com/numpy/numpy/issues/24703#issuecomment-1731269424